frontend/src/components/containerTemplates/buildings/TrashTemplateBuildingsList.vue

```diff
@@ -29,10 +29,11 @@ export default {
   },
   methods: {
     addTrashContainerBuilding: function () {
-      router.push({path: '/building/create'});
+      router.push({name: 'create_building'});
     }
   },
   async beforeMount() {
+    // TODO Add Manual

```

Comment:
@KaiHozee bedoel je ook het ophalen van alle manuals?

Reply:
Yep, maar na nader inzien kan dit beter in TrashTemplateBuildingCard gedaan worden.